core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/ERASubmissionManager.java

```diff
@@ -415,6 +415,9 @@ protected void populate(Element xml, Study s) {
       centerProjectName.setTextContent(s.getProject().getAlias());
       studyDescriptor.appendChild(centerProjectName);
 
+      Element studyAbstract = xml.getOwnerDocument().createElementNS(null, "STUDY_ABSTRACT");
+      studyDescriptor.appendChild(studyAbstract);
```

Comment:
This adds an empty (no textcontent) element to the study descriptor, was this intended? (I removed the "abstract" attribute from Study in the previous PR because it was not a database column.)